src/main/java/nextstep/subway/applicaion/dto/LineResponse.java

```diff
@@ -17,6 +17,10 @@ public LineResponse(Long id, String name, String color, LocalDateTime createdDat
         this.modifiedDate = modifiedDate;
     }
 
+    public static LineResponse of(Long id, String name, String color, LocalDateTime createdDate, LocalDateTime modifiedDate) {
```

Comment:
이런식의 정적메소드는 사실 의미가 없을 것 같지 않나요 ? 🤔
생성자에 들어가는 매개변수의 숫자가 너무 많기에 이를 줄여보려고 한 것인데 말이죠 ㅎ. 

```suggestion
    public static LineResponse of(Line line) {
```

이렇게 바꿀수는 없을까요?